packages/utilities/psammead-styles/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 1.0.2   | [PR#764](https://github.com/bbc/psammead/pull/764) Add functions to get serif bold font |
```

Comment:
Should we increment the minor version since we are adding a new method?
cc @sadickisaac @dr3 @jblaut

Reply:
Yeah we should increment the minor version since we are adding functionality, that is backward compatible and doesn't  break existing usage of `font-styles`